pkg/virtctl/hotplug/cpu.go

```diff
@@ -0,0 +1,113 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 The KubeVirt Contributors
+ *
+ */
+
+package hotplug
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+var cpuCount uint32
+
+func NewCPUCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "cpu (VMI)",
+		Short:   "Set the number of CPUs for a running virtual machine instance.",
+		Example: cpuUsage(),
+		Args:    templates.ExactArgs("cpu", 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := CPU{clientConfig: clientConfig}
+			return c.Run(args)
+		},
+	}
+
+	cmd.Flags().Uint32Var(&cpuCount, "count", 0, "Number of CPUs to set.")
```

Comment:
Does it have to be a flag or could it be a simple positional argument?

`virtctl hotplug cpu myvm 2`

Reply:
I changed it to positional argument, but notice that when it's a positional argument and the value is negative it gets interpreted as shortened flag which provides error like this 
`unknown shorthand flag: '1' in -1` for example
when you would expect some sane message like `argument must be positive`

Reply:
I see, but a negative cpu count is an error anyways.